ospd/ospd.py

```diff
@@ -1408,7 +1408,7 @@ def handle_get_version_command(self) -> str:
 
         return simple_response_str('get_version', 200, 'OK', content)
 
-    def handle_command(self, command: str) -> str:
+    def handle_command(self, command: str, stream) -> str:
```

Comment:
IMHO handle_command should not return anything anymore because we are writing the string already to the steam.

Reply:
done